menas/ui/service/EntityDialog.js

```diff
@@ -97,7 +97,32 @@ class DatasetDialog extends EntityDialog {
 
  }
 
+  /**
+   * Will created oProps's allowedValues mapped into displayable sequence of objects, e.g.
```

Comment:
Seems as a incorrect grammar in the comment _Will created..._

Reply:
True, result of re-editing, will fix, thanks.